systemtest/instrumentation_test.go

```diff
@@ -89,8 +89,10 @@ func TestAPMServerInstrumentation(t *testing.T) {
 		if !ok {
 			continue
 		}
-		assert.Equal(t, transactionDoc.Trace.ID, traceID)
-		assert.Equal(t, transactionDoc.Transaction.ID, entry.Fields["transaction.id"])
+		assert.Equal(t, transactionDoc.Trace.ID, traceID,
+			"expecting log with trace id %s; got trace id %s and message \"%s\" instead", transactionDoc.Trace.ID, traceID, entry.Message)
+		assert.Equal(t, transactionDoc.Transaction.ID, entry.Fields["transaction.id"],
```

Comment:
**suggestion**: I think we can use `assert#Equalf` for clarity. WDYT ?

Reply:
Agreed for clarity. `Equalf` uses `Equal` anyways.